CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.24)
```

Comment:
same for this file, I will delete it from master after pull request is accepted (I forgot that everything I pushed would go to the pull request)